### ISSUE TITLE
Deformable Convolution Layer

### ISSUE THREAD

**Issue by mdv3101** [tensorpack/tensorpack#1415]
Deformable Convolution operations are quite useful and has give a boost in accuracy. There are few code implementations:

[https://github.com/DHZS/tf-deformable-conv-layer](https://github.com/DHZS/tf-deformable-conv-layer). It has eager only execution available. Is there any solution to integrate with tensorpack. (specially Cascade RCNN).

Also any other suggestion to integrate Deformable Convolution?

**Comment by ppwwyyxx** [comment]
See https://tensorpack.readthedocs.io/tutorial/symbolic.html#symbolic-layers:
> Tensorpack contains a small collection of common model primitives, such as conv/deconv, fc, bn, pooling layers. However, tensorpack is model-agnostic, which means you do not need to use tensorpack’s symbolic layers and can skip this tutorial.
These layers were written only because there were no alternatives when tensorpack was first developed. Nowadays, many of these implementation actually call tf.layers directly. Tensorpack will not add any more layers into its core library because this is not the focus of tensorpack, and there are many other alternative symbolic libraries today.
Today, you can just use tf.layers or any other symbolic libraries inside tensorpack.
